test/index.js

```diff
@@ -1,17 +1,16 @@
 var assert = require('assert');
-var babel = require('babel');
+var babel = require('babel-core');
 var chalk = require('chalk');
 var clear = require('clear');
 var diff = require('diff');
 var fs = require('fs');
 
-require('babel/register');
+require('babel-register');
 
 var pluginPath = require.resolve('../src');
 
 function runTest() {
 	var output = babel.transformFileSync(__dirname + '/fixtures/actual.js', {
-		optional: ['runtime'],
 		plugins: [pluginPath]

```

Comment:
I can definitely add it if you'd like, though my personal leaning is that the test code should exercise the hello-world plugin in isolation... i.e., we just want to make sure the plugin itself transforms the code in the way we want it to, which is easiest to see if it's the only plugin active, and there's no real need for the code to also be piped through runtime/ES2015 transformers since we never actually run the transformed output (we just string diff it). Thoughts?